hangman.js

```diff
@@ -1 +1,128 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+guessNumber = 0;
+totalGuesses = 6
+correctGuesses = 0;
+remainingGuesses = totalGuesses;
+guess = "";
+word = "";
```

Comment:
can you think of a way to move the above global variables down into your game object?